.github/ISSUE_TEMPLATE/I_need_help.md

```diff
@@ -0,0 +1,33 @@
+---
+name: I need help
+about:  Write to Nisaba developers about a problem you have encountered
+
+---
+
+**Describe the problem**
+A clear and concise description of what the problem is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```

Comment:
Maybe add "e.g." or something to make it clear they don't have to be exactly these steps?